site/src/components/Filter/Filter.tsx

```diff
@@ -187,15 +189,18 @@ export const Filter: FC<FilterProps> = ({
 				display: "flex",
 				gap: 8,
 				marginBottom: 16,
-				flexWrap: "nowrap",
+				flexWrap: "wrap",
 
-				[theme.breakpoints.down(breakpoint)]: {
-					flexWrap: "wrap",
+				[theme.breakpoints.up(singleRowBreakpoint)]: {
+					flexWrap: "nowrap",
 				},
 			}}
 		>
 			{isLoading ? (
-				skeleton
+				<>
+					<BaseSkeleton width="100%" />
+					{optionsSkeleton}
+				</>
```

Comment:
Inlined the base skeleton because:
1. I don't think there will ever be a situation where we don't want a skeleton for the main search box itself
2. It felt weird that the component was never in control of _any_ of its skeletons